performance.rst

```diff
@@ -81,7 +81,7 @@ container into a single file, which could improve performance when using
         // config/services.php
 
         // ...
-        $container->parameters()->set('.container.dumper.inline_factories', true);
+        $container->getParameterBag()->set('.container.dumper.inline_factories', true);
```

Comment:
I think here `$container` is a `ContainerConfigurator` and not `ContainerInterface`

If i'm right, we should probably update example like is done in the following sample

https://github.com/symfony/symfony-docs/blob/19ca84589284485ad490b617db7c4562ba5ac4d8/configuration.rst?plain=1#L375-L383